apps/web/src/lang/fr.json

```diff
@@ -7766,6 +8534,10 @@
     "defaultMessage": "Nom de l’équipe (Anglais)",
     "description": "Label for the English team display name input"
   },
+  "n7RwgC": {
+    "defaultMessage": "Autorité indépendante non gouvernementale requise (par exemple, examen indépendant des services.)",
```

Comment:
```suggestion
    "defaultMessage": "Autorité indépendante non gouvernementale requise (p. ex. examen indépendant des services)",
```

Reply:
Response from Rosita: prefer long form in French, even when short in English